### DISCUSSION TITLE
Android support?

### DISCUSSION THREAD

**Discussion by vsod99** [OpenTabletDriver/OpenTabletDriver#1006] [General]
Would it be possible to bring android support to this driver? With the integration in osu!lazer, this could make pretty much any phone an awesome portable osu device...

**Comment by InfinityGhost** [comment]
Eventually we could add Android support, but it wouldn't be directly usable within Android itself due to the lack of any virtual device APIs and within osu!lazer instead.

**Reply by vsod99** [reply]
I figured this was the case. Would be a neat idea regardless!
